packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -1143,6 +1146,9 @@ export default class Meetings extends WebexPlugin {
           sessionId: meeting.locusInfo?.fullState?.sessionId,
         },
       },
+      roles: meeting.roles,
+      callStateForMetrics: meeting.callStateForMetrics,
+      getCurUserType: meeting.getCurUserType,
```

Comment:
initially I didn't like this approach, because I thought that having a reference to the method of a meeting will prevent that meeting object from being garbage collected, but I think it's OK, you can have references to methods of an object and these methods on their own don't have a reference to the object itself (as long as they don't use any closures), so it should be fine.